src/components/Refresh/Refresh.tsx

```diff
@@ -0,0 +1,66 @@
+import { Store } from 'babydux'
+import { chain } from 'lodash'
+import { RaisedButton } from 'material-ui'
+import { store, withStore, Actions } from '../../services/store'
+import { getAdequacies } from  '../../services/api'
+import { Measure, Standard } from '../../constants/datatypes'
+import { TIME_DISTANCES } from '../../constants/timeDistances'
+import * as React from 'react'
+import './Refresh.css'
+
+
+function isAdequate(
+  distance: number,
+  time: number,
+  measure: Measure,
+  standard: Standard
+) {
+  switch (standard) {
+    case 'distance':
+      return distance < measure
+    case 'time_distance':
+      return distance < measure && time < TIME_DISTANCES.get(measure)!
+    case 'time':
+      return time < TIME_DISTANCES.get(measure)!
+  }
+}
+
+async function recomputeAdequacies(store: Store<Actions>) {
```

Comment:
This is an effect, and should go in effects.ts. If you're having trouble triggering it, dispatch an Action (ie. `store.set(..)(..)` from the `onClick` below.